projects/step-frontend/src/lib/modules/timeseries/chart/tooltip-plugin.ts

```diff
@@ -169,6 +169,12 @@ export class TooltipPlugin {
           bound = over;
           //	bound = document.body;
 
+          over.onclick = () => {
+            // @ts-ignore
+            const lockState = u.cursor._lock;
+            ref.onLockStateChange.next(lockState);
```

Comment:
Even `EventEmitter` is the `Subject`'s descendant, it make sense to follow the `EventEmitter` api and call `emit` instead of `next`, even both methods are doing the same